### DISCUSSION TITLE
Problems in laravel 11

### DISCUSSION THREAD

**Discussion by itsleviaa10** [laravel/framework#53765] [Q&A] [answered]
This is my first time using Laravel 11. Here I want to ask whether the middleware folder is removed in Laravel 11.

**Comment by didikz** [comment, marked as answer]
Yes it's already included on framework level now. Please refer this release note [https://laravel.com/docs/11.x/releases#laravel-11](https://laravel.com/docs/11.x/releases#laravel-11)

**Reply by itsleviaa10** [reply]
Thank you bro
